packages/ERTP/src/issuer.js

```diff
@@ -178,7 +183,10 @@ function makeIssuerKit(
     const newTotal = newPaymentBalances.reduce(add, empty);
 
     // Invariant check
-    assert(amountMath.isEqual(total, newTotal), 'rights were not conserved');
+    assert(
+      amountMath.isEqual(total, newTotal, brand),
```

Comment:
Less compelling but still pleasant to factor out `isEqual` as well.
```suggestion
      isEqual(total, newTotal),
```